pdfplumber/cli.py

```diff
@@ -28,6 +28,8 @@ def parse_args(args_raw):
 
     parser.add_argument("--laparams", type=json.loads)
 
+    parser.add_argument("--precision", type=int)
```

Comment:
The check should be for positive integers only. With just `int`, negative numbers will be accepted as well. Reference for implementation: https://stackoverflow.com/a/14117511/7760998

Reply:
Hmmm, interesting! It may counterintuitive and hard to imagine a great use-case, but negative values seem, at least to me, that they could be legitimate for `precision`, since negative rounding is possible in Python. I.e.,:

```python
for i in reversed(range(-2, 3)):
    print(f"{i: >2}: {round(777.777, i):.3f}")
```

```
 2: 777.780
 1: 777.800
 0: 778.000
-1: 780.000
-2: 800.000
```

My instinct is to allow negative values. What do you think?

Reply:
Oh, I didn't know you could set negative values in the round operation. Makes sense then. Thanks.